detections/cloud/azure_ad_admin_consent_bypassed_by_service_principal.yml

```diff
@@ -16,16 +16,17 @@ description: The following analytic identifies instances where a service princip
   activity could allow attackers to exploit automation to assign sensitive permissions
   without proper oversight, potentially compromising the security of the Azure AD
   environment.
-search: "`azure_monitor_aad` (operationName=\"Add app role assignment to service principal\"\
-  \ OR operationName=\"Add member to role*\") src_user_type=servicePrincipal  | rename
-  properties.* as *   | eval roleId = mvindex('targetResources{}.modifiedProperties{}.newValue',
-  0)  | eval roleValue = mvindex('targetResources{}.modifiedProperties{}.newValue',
-  1)  | eval roleDescription = mvindex('targetResources{}.modifiedProperties{}.newValue',
-  2)  | eval user_id = mvindex('targetResources{}.id', 0), user=coalesce(user,mvindex('targetResources{}.displayName',
-  0)) | rename initiatedBy.app.displayName as src_user  | stats count earliest(_time)
-  as firstTime latest(_time) as lastTime by src_user user user_id roleId roleValue
-  roleDescription | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)`\
-  \  | `azure_ad_admin_consent_bypassed_by_service_principal_filter`"
+search: "`azure_monitor_aad` (operationName=\"Add app role assignment to service principal\" OR operationName=\"Add member to role*\") src_user_type=servicePrincipal 
+  | rename properties.* as * 
+  | eval roleId = mvindex('targetResources{}.modifiedProperties{}.newValue',0) 
+  | eval roleValue = mvindex('targetResources{}.modifiedProperties{}.newValue',1) 
+  | eval roleDescription = mvindex('targetResources{}.modifiedProperties{}.newValue',2) 
+  | eval user_id = mvindex('targetResources{}.id', 0), user=coalesce(user,mvindex('targetResources{}.displayName',0))
+  | rename initiatedBy.app.displayName as src_user, operationName as action, userAgent as user_agent
+  | fillnull 
```

Comment:
let not rename operationName as action, operationName is already aliased to signature which can be included in the output along with signature

Reply:
good idea to use cim signature as output